charts/prometheus-elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.12.0
+version: 4.12.1
```

new feature shoudl raise minor version.
```suggestion
version: 4.13.0
```